neuralmonkey/config/builder.py

```diff
@@ -163,7 +170,9 @@ def instantiate_class(name, all_dicts, existing_objects, depth):
     return obj
 
 
-def build_config(config_dicts, ignore_names, warn_unused=False):
+def build_config(config_dicts: Dict[str, Any],
+                 ignore_names: Set[str],
+                 warn_unused=False) -> Dict[str, Any]:
```

Comment:
Máš pravdu. Sorry, jsem zvyklej na type systémy, které mají větší než nulovou vyjadřovací sílu. V čemkoliv rozumném je z toho, že něco instanciuješ hodnotou nějakého typu možné odvodit, že ta proměnná má ten typ. Ale to by nebyl Python, aby v tomhle nedokázal zavést víceznačnost.